docs/en/datasets/recog.md

```diff
@@ -79,6 +79,11 @@
 │   │   ├── annotations
 │   │   ├── train_label.txt
 │   │   ├── test_label.txt
+│   ├── IIIT-ILST
+│   │   ├── crops
+│   │   ├── ignores
+│   │   ├── train_label.txt
+│   │   ├── test_label.txt
```

Comment:
default format is `jsonl`, so it should be `train_label.jsonl` and `test_label.jsonl` here.

Reply:
ok